hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServer.java

```diff
@@ -146,6 +147,19 @@ public Void call() throws Exception {
       reconSchemaManager.createReconSchema();
       LOG.debug("Recon schema creation done.");
 
+      LOG.info("Finalizing Layout Features.");
+      // Handle Recon Schema Versioning
+      ReconSchemaVersionTableManager versionTableManager =
+          injector.getInstance(ReconSchemaVersionTableManager.class);
+      DataSource dataSource = injector.getInstance(DataSource.class);
+
+      ReconLayoutVersionManager layoutVersionManager =
+          new ReconLayoutVersionManager(versionTableManager, reconContext, dataSource);
+      // Run the upgrade framework to finalize layout features if needed
+      layoutVersionManager.finalizeLayoutFeatures();
+
+      LOG.debug("Recon schema versioning completed.");
```

Comment:
We can keep this as `INFO`, as this will not repeatedly log, so no issues.

Reply:
Done, please take a look